db/migrate/20190212211753_create_timelines.rb

```diff
@@ -2,7 +2,7 @@ class CreateTimelines < ActiveRecord::Migration[5.1]
   def change
     create_table :timelines do |t|
       t.string :title
-      t.references :user, foreign_key: true, type: :integer
+      t.references :user, foreign_key: true, type: :bigint
```

Comment:
Rails 4.x to 5.x changed autogenerated ID columns default type from integer to bigint. I think this was causing Travis failure.  Note: PR #627 (Upgrade step 5) proposes a solution in the context of preexisting data.